tests/unit_tests/test_wave_01.py

```diff
@@ -40,16 +40,21 @@ def test_removing_from_inventory_returns_item():
     assert item not in vendor.inventory
     assert result == item
 
-@pytest.mark.skip
+# @pytest.mark.skip
 def test_removing_not_found_is_false():
     item = "item to remove"
     vendor = Vendor(
         inventory=["a", "b", "c"]
     )
 
-    result = vendor.remove(item)
+    result = vendor.remove("d")
```

Comment:
Please do not modify parts of the test that you are not supposed to modify. The instructions say to complete the Assert portion of the test. You should not be modifying this part.